chapter_11.md

```diff
@@ -1,16 +1,202 @@
 Visual Regression Testing
 =========================
 
-We now finished our basic test framework setup and can now look into adding more testing capabilities to it. The advantage of using the WebdriverIO testrunner is that in nicely integrates with cloud services like Sauce Labs or Applitools. In this chapter we now want to focus on building an integration with Applitools so that we can add a visual regression testing component to our tests.
+We now finished our basic test framework setup and can now look into adding more testing capabilities to it. The advantage of using the WebdriverIO testrunner is that in nicely integrates with all kinds of services. One of them is the [`@wdio/visual-service`](https://github.com/webdriverio/visual-testing). This service is specifically made for WebdriverIO and can be used for:
 
-1. Create a free trial account on [applitools.com](https://applitools.com/)
-2. Add the [`@wdio/applitools`](https://www.npmjs.com/package/@wdio/applitools-service) service to your config that runs tests on Sauce Labs
-3. Add your Applitools key to your config
-4. Enhance your existing tests so that it takes a screenshot after each test
-5. __Bonus:__ Modify the test so that it functionally still passes but visual not
+- 🖥️ Desktop browsers (Chrome / Firefox / Safari / Microsoft Edge)
+- 📱 Mobile / Tablet browsers (Chrome on Android emulators / Safari on iOS Simulators / Simulators / real devices) via Appium
+- 📱 Native Apps (Android emulators / iOS Simulators / real devices) via Appium
+- 📳 Hybrid apps via Appium
+- 📖 Storybook (BETA)
 
-Depending on the scenario we are usually only interested to run our visual regression tests in CI/CD. It wouldn't make much sense to also include the service into our local config, since our developers or QA engineers might work with the applications while running e2e tests. This could break our baseline.
+The goals for Visual Testing will be to create
 
-You can find all information on how to take snapshots within the test in the documentation of the service.
+- Desktop element/viewport/fullpage snapshot tests
+- Native Mobile app element/viewport snapshots
+- and if time lets us, create storybook snapshots
 
-__Tipp:__ You can use the [`execute`](https://webdriver.io/docs/api/browser/execute.html) command in order to inject JavaScript to modify the page visually.
+## Setup
+
+1. Create a sample folder with `mkdir visual-demo` and `cd` into it
```

Comment:
Previous chapters built onto the existing code base. Do you think this makes sense for visual? If not we should clearly state that we start a new project.

Reply:
You are right, will highlight that at the beginning